volumes/provisioner/jiva/snapshot_revert.go

```diff
@@ -0,0 +1,38 @@
+package jiva
+
+import (
+	"fmt"
+
+	"github.com/openebs/maya/command"
+)
+
+func (c *ControllerClient) RevertSnapshot(volname string, snapshot string) error {
+
+	annotations, err := command.GetVolumeSpec(volname)
+	if err != nil || annotations == nil {
+
+		return err
+	}
+
+	if annotations.ControllerStatus != "Running" {
```

Comment:
Use typed constants

Reply:
I will do this, this should be typed constant. Thanks